pkg/generated/clientset/versioned/clientset.go

```diff
@@ -59,7 +59,7 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
 	configShallowCopy := *c
 	if configShallowCopy.RateLimiter == nil && configShallowCopy.QPS > 0 {
 		if configShallowCopy.Burst <= 0 {
-			return nil, fmt.Errorf("Burst is required to be greater than 0 when RateLimiter is not set and QPS is set to greater than 0")
+			return nil, fmt.Errorf("burst is required to be greater than 0 when RateLimiter is not set and QPS is set to greater than 0")
```

Comment:
Is this not autogenerated code? Not sure this should be modified manually to fix linting errors

Reply:
right, it gets rewritten by `make generate`